Eduria/Eduria/Controllers/UserEQLogController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Eduria.Services;
+using EduriaData.Models;
+using EduriaData.Models.ExamLayer;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Eduria.Controllers
+{
+    public class UserEQLogController
+    {
+        private UserEQLogService _userEQLogService;
+
+        public UserEQLogController(UserEQLogService userEqLogService)
+        {
+            _userEQLogService = userEqLogService;
+        }
+
+        public void AddUserEqLog(UserEQLog userEqLog)
```

Comment:
Deze method kan in principe weg:

En dan bij AddListOfUserEqLogs:
```suggestion
        public void AddListOfUserEqLogs(List<UserEQLog> userEqLogs) 
        {
              _userEQLogService.Add(userEqLog);
        }
```